test/auto_parallel/pir/test_relu_spmd_rule.py

```diff
@@ -0,0 +1,104 @@
+# Copyright (c) 2024 PaddlePaddle Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+
+import paddle
+import paddle.distributed as dist
+from paddle.distributed.auto_parallel.api import dtensor_from_local
+
+paddle.enable_static()
+
+
+class TestReluSpmdRule(unittest.TestCase):
+    def test_build_replicated_program(self):
+        main_program = paddle.base.Program()
+        with paddle.base.program_guard(main_program):
+            mesh = dist.ProcessMesh([0, 1], dim_names=['mp'])
+            x = paddle.static.data(name='x', shape=[64, 36])
+            dist_x = dtensor_from_local(x, mesh, [dist.Replicate()])
```

Comment:
better use shard_tensor to construct dist_tensor

Reply:
done, thanks!